src/main/resources/edu/cornell/kfs/fp/cu-spring-fp.xml

```diff
@@ -664,4 +672,93 @@
+    <bean id="corporateBilledCorporatePaidLoadFlatFileStep-parentBean" abstract="true" class="edu.cornell.kfs.fp.batch.CorporateBilledCorporatePaidLoadFlatFileStep" parent="step">
+        <property name="procurementCardLoadTransactionsService">
+            <ref bean="corporateBilledCorporatePaidLoadTransactionsService"/>
+        </property>
+        <property name="batchInputFileService">
+            <ref bean="batchInputFileService"/>
+        </property>
+        <property name="corporateBilledCorporatePaidInputFileType">
+            <ref bean="corporateBilledCorporatePaidFlatInputFileType"/>
+        </property>
+        <property name="reportWriterService" ref="corporateBilledCorporatePaidLoadReportWriterService"/>
+    </bean>
+    
+    <bean id="corporateBilledCorparatePaidCreateDocumentsStep" class="org.kuali.kfs.fp.batch.ProcurementCardCreateDocumentsStep" parent="step">
+        <property name="procurementCardCreateDocumentService">
+            <ref bean="corporateBilledCorporatePaidCreateDocumentService"/>
+        </property>
+    </bean>
+    
+    <bean id="corporateBilledCorporatePaidRouteDocumentsStep" class="org.kuali.kfs.fp.batch.ProcurementCardRouteDocumentsStep" parent="step">
+        <property name="procurementCardCreateDocumentService">
+            <ref bean="corporateBilledCorporatePaidCreateDocumentService"/>
+        </property>
+    </bean>
+    
+    <bean id="corporateBilledCorporatePaidFlatInputFileType" class="edu.cornell.kfs.fp.batch.CorporateBilledCorporatePaidFlatInputFileType"> 
+        <property name="directoryPath">
+           <value>${staging.directory}/fp/cbcp</value>
+        </property>
+        <property name="fileExtension">
+           <value>data</value>
+        </property>
+        <property name="parameterService">
+            <ref bean="parameterService"/>
+        </property>
+        <property name="procurementCardErrorEmailService">
+            <ref bean="corporateBilledCorporatePaidErrorEmailService"/>
+        </property>
+        <property name="dateTimeService">
+           <ref bean="dateTimeService"/>
+        </property>
+    </bean>
+    
+    <bean id="corporateBilledCorporatePaidErrorEmailService" class="edu.cornell.kfs.fp.batch.service.impl.CorporateBilledCorporatePaidErrorEmailServiceImpl">
+        <property name="parameterService">
+            <ref bean="parameterService"/>
+        </property>
+        <property name="emailService">
+            <ref bean="emailService"/>
+        </property>
+    </bean>
+    
+    <bean id="corporateBilledCorporatePaidLoadTransactionsService"  parent="corporateBilledCorporatePaidLoadTransactionsService-parentBean"/> 
+    <bean id="corporateBilledCorporatePaidLoadTransactionsService-parentBean" class="edu.cornell.kfs.fp.batch.service.impl.ProcurementCardLoadFlatTransactionsServiceImpl" parent="procurementCardLoadTransactionsService-parentBean" >
```

Comment:
Should the parent of this CBCP bean be the actual PCard service bean and not its "parentBean" variant? Also, this CBCP parent bean should be declared as abstract for consistency with this convention.